core/common/src/main/java/alluxio/security/authentication/AuthType.java

```diff
@@ -46,6 +46,11 @@
   ;
 
   /**
+   * Gets the string value of the authentication type
```

Comment:
Forgot to end the sentence with a dot (.)

Reply:
Fixed.